.env.example

```
@@ -28,6 +28,13 @@ PUBLIC_DGEN_NETWORK=mainnet  # Options: mainnet, testnet, regtest
 PUBLIC_EXPLORER=https://blockstream.info
 PUBLIC_LIQUID_EXPLORER=https://blockstream.info/liquid
 
+# Breez SDK Esplora Overrides (OPTIONAL)
+# Use the backend proxy so the paid Blockstream key is applied server-side.
+# Keep Liquid unset to allow Breez SDK to use its default Liquid explorer (recommended).
+# Production: https://your-api.railway.app/api/esplora/bitcoin
+VITE_BITCOIN_EXPLORER_URL="http://localhost:3119/api/esplora/bitcoin"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)